Live/Classes/GameTimer.cpp

```diff
@@ -0,0 +1,30 @@
+#include "GameTimer.h"
+
+GameTimer::GameTimer() {
+  start_time = Clock::now();
+}
+
+GameTimer::~GameTimer() {}
+
+int GameTimer::getTime() {
+  return std::chrono::duration_cast<std::chrono::seconds>(Clock::now()-start_time).count();
+}
+
+int GameTimer::getDays() {
+  return getTime()/10;
+}
+
+GameTimer::DayPhase GameTimer::getCurrentDayPhase() {
+  if(getTime()/5 % 2 == 0) {
+    return DayPhase::DAY;
+  }
+  return DayPhase::NIGHT;
+}
+
+GameTimer* GameTimer::instance() {
+  if (!GameTimer::shared)
+    GameTimer::shared = new GameTimer();
+  return GameTimer::shared;
+}
+
+GameTimer* GameTimer::shared = nullptr;
```

Comment:
Please watch for the two space indentations, fix your text editors if you don't wanna manually deal with it.